### PR TITLE
Update argument handling

### PR DESCRIPTION
**Pull request by tysmith**

### CODE REVIEW

File: grizzly/args.py [hunk 1]
```diff
@@ -52,14 +51,16 @@ def __init__(self):
...
         self.parser.add_argument("binary", help="Firefox binary to run")
         self.parser.add_argument(
             "--log-level",
+            choices=sorted(self._level_map),
             default="INFO",
-            help="Configure console logging. Options: %s (default: %%(default)s)"
-            % ", ".join(
-                k for k, v in sorted(self._level_map.items(), key=lambda x: x[1])
-            ),
+            help="Configure console logging (default: %(default)s)",
```
Comment (jschwartzentruber) [inline grizzly/args.py:63]: You could use `"Configure console logging. Options: %(choices)s (default: %(default)s)"` to keep the old format.

File: grizzly/args.py [hunk 2]
```diff
@@ -109,8 +110,8 @@ def __init__(self):
         self.launcher_grp.add_argument(
             "--platform",
             default="ffpuppet",
-            help="Installed Platforms (Targets): %s (default: %%(default)s)"
-            % ", ".join(sorted(scan_plugins("grizzly_targets"))),
+            choices=sorted(targets),
+            help="Target to use (default: %(default)s)",
```
Comment (jschwartzentruber) [inline grizzly/args.py:114]: Ditto

**Review by jschwartzentruber — APPROVED** [review]
Looks good as is. Just one comment in two places.

File: grizzly/args.py (hunk 1 above)
Comment (tysmith) [reply grizzly/args.py:63]: But wouldn't that be redundant?

We now get the choices automatically:
```
--log-level {CRIT,DEBUG,ERROR,INFO,WARN}
                      Configure console logging (default: INFO)
```

Comment (jschwartzentruber) [reply grizzly/args.py:63]: you're right

**Pull request merged**